src/main/java/org/sonarlint/intellij/cayc/HelpLabel.kt

```diff
@@ -21,15 +21,27 @@ package org.sonarlint.intellij.cayc
 
 import com.intellij.ide.BrowserUtil
 import com.intellij.ui.ContextHelpLabel
+import org.sonarlint.intellij.common.util.SonarLintUtils
 import org.sonarlint.intellij.documentation.SonarLintDocumentation
+import org.sonarlint.intellij.telemetry.LinkTelemetry
+import org.sonarlint.intellij.telemetry.SonarLintTelemetry
 
-class FocusModeHelpLabel {
+class HelpLabel {
     companion object {
         @JvmStatic
-        fun create() =
+        fun createCleanAsYouCode() =
                 ContextHelpLabel.createWithLink(null,
                         "Deliver clean code by focusing on code that was recently modified",
-                        "Learn more about Clean as You Code", true) { BrowserUtil.browse(SonarLintDocumentation.FOCUS_ON_NEW_CODE_LINK) };
+                        "Learn more about Clean as You Code", true) { BrowserUtil.browse(SonarLintDocumentation.FOCUS_ON_NEW_CODE_LINK) }
+
+        @JvmStatic
+        fun createConnectedMode() =
+            ContextHelpLabel.createWithLink(null,
+                "Unleash the full potential of SonarLint in Connected Mode with SonarQube or SonarCloud to analyze more languages, detect more issues, and receive notifications about the quality gate status.",
+                "Learn more about Connected Mode", true) {
+                SonarLintUtils.getService(SonarLintTelemetry::class.java).addQuickFixAppliedForRule(LinkTelemetry.CONNECTED_MODE_DOCS.linkId)
```

Comment:
Same here with `addQuickFixAppliedForRule`, I suggest reviewing all the calls to telemetry to make sure we don't misuse this elsewhere

Reply:
In fact those 2 lines are duplicated in several places, so why not having a class taking care of those 2 actions and that would be invoked from several places?

Reply:
For all those links, the action is to update the telemetry and open the URL in the browser. My suggestion is just to factorize those 2 calls in a single place. It's more a nice-to-have